pywbem/config.py

```diff
@@ -60,3 +61,11 @@
 #: operations.
 
 DEFAULT_ITER_MAXOBJECTCOUNT = 1000
+
+#: Maximum log entry size. An integer that sets the maximum size of each log
+#: entry if the log_detail_level flag is set.
+
+DEFAULT_MAX_LOG_ENTRY_SIZE = 1000
+
+#: Set default log destination.  None means that there will be no logging
+DEFAULT_LOG_DESTINATION = 'file'
```

Comment:
Why do we have `DEFAULT_LOG_DESTINATION` here in `config.py`, but `DEFAULT_LOG_DETAIL_LEVEL` and `DEFAULT_LOG_DETAIL_LEVEL` are in `_logger.py`?